setup.py

```diff
@@ -70,20 +53,29 @@ def run(self):
         super().run()
 
 
-# Only include pytest-runner in setup_requires if we're invoking tests
-if {'pytest', 'test', 'ptr'}.intersection(sys.argv):
-    setup_requires = ['pytest-runner']
-else:
-    setup_requires = []
+install_requires = []
+tests_require = [
+    'pytest>=5',
+    'pytest-env'
```

Comment:
Removed `pytest-cov` as it isn't a testing dependency, it is only needed for codecov upload.